install/kubernetes/helm/istio/charts/mixer/templates/configmap.yaml

```diff
@@ -11,3 +12,4 @@ metadata:
     istio: mixer
 data:
   mapping.conf: |-
+{{- end }}
```

Comment:
![image](https://user-images.githubusercontent.com/4461983/44127912-5151a20e-a072-11e8-8a4c-14f8ff699ed5.png)
Add a new line to fix this?